src/main/java/domain/RaceNumber.java

```diff
@@ -0,0 +1,14 @@
+package domain;
+
+public class RaceNumber {
+    private final int raceNumber;
+    public RaceNumber(int raceNumber) {
+        if (raceNumber < 0)
+            throw new RuntimeException("CarNumber can not be negative");
```

Comment:
RaceNumber 가 되어야 겠네요 😄 
```suggestion
            throw new RuntimeException("RaceNumber can not be negative");
```